charts/dex.yaml

```diff
@@ -0,0 +1,102 @@
+image: '{%- endraw -%}{{ build_image_name(\"dex\", False) }}{%- raw -%}'
+
+nodeSelector:
+  node-role.kubernetes.io/master: ''
+
+tolerations:
+  - key: "node-role.kubernetes.io/bootstrap"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/master"
+    operator: "Exists"
+    effect: "NoSchedule"
+  - key: "node-role.kubernetes.io/infra"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+replicas: 2
+
+# grpc support
+grpc: false
+
+# https termination by dex itself
+https: false
```

Comment:
Why not?

Reply:
Indeed, even if we expose Dex through an Ingress object, it would make sense to still have it secured and exposed using a `NodePort` (though not entirely sure about that part, cc @NicolasT), as we do for K8s and Salt APIs.

Reply:
No no no, we don't want/need a `NodePort`.

However, when we *can* secure in-cluster over-the-wire traffic, we should.

Reply:
OK OK OK 😄 No `NodePort` then!

Reply:
Yeah,  I am going to enable TLS here